agent/command/git_test.go

```diff
@@ -346,7 +346,7 @@ func (s *GitGetProjectSuite) TestTokenScrubbedFromLogger() {
 	foundCloneErr := false
 	for _, msgs := range s.comm.GetMockMessages() {
 		for _, msg := range msgs {
-			if strings.Contains(msg.Message, "https://[redacted oauth token]:x-oauth-basic@github.com/deafgoat/doesntexist.git") {
+			if strings.Contains(msg.Message, "https://[redacted oauth token]:x-oauth-basic@github.com/evergreen-ci/sample.git") {
```

Comment:
Oh interesting; quite possible this test doesn't actually need this info to match and it was just a copy paste, I'll update though (good on you for reading the tests so carefully !!)

Reply:
Yeah, it's a little odd that this test passes regardless of whether you use `sample` or `doesntexist`, but I guess it's not that important to the test since it's testing that the GitHub token is redacted.